### PR TITLE
[BLOCKED]ebooks: rename electronic terms to ebook

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
Depends on inveniosoftware/invenio-app-ils/pull/1065

- removed some TODO comments

electronic medium -> e-book
electronic item -> e-item
electronic copy -> digital copy
online resources -> e-resources
electronic volumes description -> e-journal description

Closes CERNDocumentServer/cds-ils#358

### CODE REVIEW

File: src/lib/pages/backoffice/EItem/EItemDetails/EItemHeader.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class EItemHeader extends Component {
     const { data } = this.props;
     const recordInfo = (
       <>
-        <label className="muted">Electronic copy</label> {data.metadata.pid}{' '}
+        <label className="muted">E-copy</label> {data.metadata.pid}{' '}
```
Comment (ntarocco) [inline src/lib/pages/backoffice/EItem/EItemDetails/EItemHeader.js:18]:
```suggestion
        <label className="muted">E-item</label> {data.metadata.pid}{' '}
```

**Review by ntarocco — COMMENTED** [review]
Honestly I am a bit in doubt here. I would personally use:
* E-book wherever we are displaying that there is an e-book
* Digital version (instead of electronic) when we have for example the sentence `Request a new electronic copy` -> `Request the digital version`
* E-item in the backoffice, that's ok

Personally, I don't like the `e-copy/e-volume` terms, not sure what that would mean.

WDYT? Ping @jrcastro2 @kprzerwa

File: cypress/integration/frontsite/homepage.test.spec.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ describe('frontsite homepage', () => {
       .and('include', 'sort=created')
       .and('include', 'order=desc')
       .and('include', 'doctype%3ABOOK')
-      .and('include', 'medium%3AELECTRONIC_VERSION');
+      .and('include', 'medium%3AE-BOOK');
```
Comment (kprzerwa) [inline cypress/integration/frontsite/homepage.test.spec.js:21]: did you check if this works everywhere with the hyphen? Aggregations and searches mainly?

**Review by kprzerwa — COMMENTED** [review]
> Honestly I am a bit in doubt here. I would personally use:
> 
> * E-book wherever we are displaying that there is an e-book
> * Digital version (instead of electronic) when we have for example the sentence `Request a new electronic copy` -> `Request the digital version`
> * E-item in the backoffice, that's ok
> 
> Personally, I don't like the `e-copy/e-volume` terms, not sure what that would mean.
> 
> WDYT? Ping @jrcastro2 @kprzerwa

I agree. In general we should look for a balance between the label being understandable and consistent with the rest. e-book and maybe e-resource are OK in my opinion but the rest it maybe going too "specifiic"
`Request the digital version` sounds quite nice.

File: cypress/integration/frontsite/homepage.test.spec.js (hunk 2 above)
Comment (KonstantinaStoikou) [reply cypress/integration/frontsite/homepage.test.spec.js:21]: yes it works

File: src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/DocumentEItems.js [hunk 3]
```diff
@@ -31,10 +31,8 @@ export class DocumentEItems extends Component {
           <DocumentEItemUrls eitems={eitems} />
         ) : (
           <>
-            No electronic resources currently available.{' '}
-            <Link to={this.onClickEItemRequestLink()}>
-              Request a new electronic copy
-            </Link>
+            No e-resources currently available.{' '}
+            <Link to={this.onClickEItemRequestLink()}>Request the ebook</Link>
```
Comment (ntarocco) [inline src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/DocumentEItems.js:35]:
```suggestion
            <Link to={this.onClickEItemRequestLink()}>Request the e-book</Link>
```

File: src/lib/pages/backoffice/EItem/EItemDetails/EItemHeader.js [hunk 4]
```diff
@@ -15,7 +15,7 @@ export class EItemHeader extends Component {
     const { data } = this.props;
     const recordInfo = (
       <>
-        <label className="muted">Electronic copy</label> {data.metadata.pid}{' '}
+        <label className="muted">Digital copy</label> {data.metadata.pid}{' '}
```
Comment (ntarocco) [inline src/lib/pages/backoffice/EItem/EItemDetails/EItemHeader.js:18]:
```suggestion
        <label className="muted">E-item</label> {data.metadata.pid}{' '}
```

**Review by ntarocco — APPROVED** [review]

**Pull request merged**